sdcm/cluster.py

```diff
@@ -5223,7 +5223,7 @@ def start_scylla_monitoring(self, node):
             cd -P {self.monitor_install_path}
             mkdir -p {self.monitoring_data_dir}
             echo "" > UA.sh
-            ./start-all.sh \
+            PATH=$PATH:/usr/sbin ./start-all.sh \
```

Comment:
did you mean to have it added in a line before `./start-all.sh \`, or is it intentionally in the same line (i don't think it would work)

Reply:
It is intentionally on the same line. Such syntax means that an env variable defined for the following command only